src/main/resources/CompetitionRequests.sql

```diff
@@ -0,0 +1,52 @@
+---- Вывод итоговой таблицы (общий протокол) со всеми данными
+SELECT
+    d.meters || ' ' || d.name_style AS distance,
+    s.surname,
+    s.first_name,
+    s.date_birth,
+    c.name_city ,
+    p.result_time,
+    comp.name_competitions,
+    p.points
+FROM
+    Protokol p
+JOIN
+    Distance d ON p.id_distance = d.id_distance
+JOIN
+    Sportsmans s ON p.id_sportsman = s.id_sportsman
+JOIN
+    Competitions comp ON p.id_competition = comp.id_competitions
+JOIN
+    City c ON s.city_id = c.id
+ORDER BY result_time
+
+
+--- Топ 3 спортсмена, которые набрали больше всего очков на данных соревнованиях
```

Comment:
you select top 3 athletes, who get more points on Race. But should display among all races. One athlete could swim only 1 race and get 30 points.Another athlete could swim 3 races and get 10, 20, 20 points. In your select it will print first athlete.